orc/src/main/java/org/apache/iceberg/orc/ORC.java

```diff
@@ -220,6 +224,10 @@ private static class Context {
       private final CompressionKind compressionKind;
       private final CompressionStrategy compressionStrategy;
 
+      private final String bloomFilterColumns;
+
+      private final double bloomFilterFpp;
```

Comment:
nit: shall we remove the blank lines in L226 and L228 so `bloomFilterColumns` and `bloomFilterFpp` will be grouped together with the rest of the properties?

Reply:
+1 to that. None of the other private fields have blank lines.